cli/pcluster/cfnconfig.py

```diff
@@ -111,6 +111,11 @@ def __init__(self, args):
         except AttributeError:
             pass
 
+    @staticmethod
+    def __warn(message):
```

Comment:
it would be nice to use the logging module to log to stdout but we can leave it as is for now

Reply:
Yeah we can leave as is for now